src/config.json

```diff
@@ -12,7 +12,9 @@
       "//2": "from bolt-0002, max channel size is 2^24 - 1 satoshis",
       "maxChannelBalance": "16777215",
       "//3": "LND specifies max payment size as UInt32 - 1 millisatohis (https://github.com/lightningnetwork/lnd/blob/v0.5.1-beta/rpcserver.go#L49-L58)",
-      "maxPaymentSize": "4294967"
+      "maxPaymentSize": "4294967",
+      "//4": "LND static channel backup file path",
```

Comment:
I'm not familiar with the config file. What's the purpose of prepending `//` on certain properties?

Reply:
@bradleystachurski its the way we designate comments. When the file is loaded. the property still exists however it was the easiest way for us to document some of the variables